PKHeX/Resources/text/es/lang_es.txt

```diff
@@ -799,3 +799,13 @@ B_Output = Exportar
 B_Cancel = Cancelar
 B_Save = Guardar
 ! End
+! -----------------------------------------------------
+- DO NOT CHANGE THIS SECTION.
+! ErrorWindow = Error
+- Change stuff below this line, not above.
+-------------------------------------------------------
+L_ProvideInfo = Please provide this information when reporting this error: 
```

Comment:
I can provide the Spanish translation for this. I'll make a PR when this gets merged, or here it is if you want to include it in this one:

L_ProvideInfo = Por favor, proporciona esta información cuando reportes este fallo:
B_CopyToClipboard = Copiar al portapapeles
B_Continue = Continuar
B_Abort = Abortar